platform/qemu.go

```diff
@@ -545,7 +545,8 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 		panic("Ignition specified but no primary disk")
 	}
 
-	if builder.Swtpm {
+	// TPM devices aren't on s390x/ppc64le at least
+	if builder.Swtpm && builder.Board == "amd64-usr" {
```

Comment:
Should we also plumb through the setting of `builder.Swtpm` to the value of `qc.flight.opts.Swtpm` in [the cluster code](https://github.com/coreos/mantle/blob/master/platform/machine/unprivqemu/cluster.go#L93) to allow for not using swtpm on x86_64 machines?

Reply:
i think the Swtpm option can be removed now that we have this

Reply:
Right sorry, I hadn't forgotten your earlier comment and I meant to fix that too with this.  Done now!